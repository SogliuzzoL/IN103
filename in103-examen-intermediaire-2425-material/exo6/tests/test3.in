19
w
e
r
c
m
o
a
g
t
n
l
l
z
j
l
m
m
a
m
i
d
a
h
a
l
n
a
r
i
a
e
a
a
u
a
a
n
i
l
o
p
a
t
i
t
b
i
c
u
o
c
c
d
t
x
t
c
l
u
h
r
t
v
o
r
s
o
r
n
h
k
o
h
e
h
h
i
a
a
l
h
i
i
i
t
l
e
a
a
s
v
i
n
o
a
a
r
e
e
e
e
n
e
a
a
n
r
r
o
i
a
c
n
e
m
d
l
s
w
r
e
l
n
s
t
d
y
n
c
s
e
y
l