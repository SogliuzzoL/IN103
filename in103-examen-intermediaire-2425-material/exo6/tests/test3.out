william
edouard
raphael
charles
matthew
olivier
antoine
gabriel
tristan
nicolas
laurent
leonard
zachary
jackson
ludovic
mathias
maxence
anthony
michael
